packages/sw-background-sync-queue/src/lib/background-sync-queue.js

```diff
@@ -129,6 +143,19 @@ class BackgroundSyncQueue {
 	replayRequests() {
 		return this._requestManager.replayRequests();
 	}
+
+  /**
+	 * sets the dbName, which is used to store the queue and requests
+	 * defaults to bgQueueSyncDB
+	 * @param {String} dbName
```

Comment:
this should be ID.

Should this method be private? If so, could you make it start with an underscore so it's more obvious it's not public,

I'm assuming it should be public as this used to be exposed by the module.